pkgs/development/python-modules/molecule-docker/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, docker
+, requests
+, molecule
+}:
+
+buildPythonPackage rec {
+  pname = "molecule-docker";
+  version = "0.3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "dc9a8ad60b70ede303805cd6865deb5fb9c162e67ff5e7d1a45eb7e58cd36b88";
+  };
+
+  propagatedBuildInputs = [
+    docker
+    molecule
+    requests
+  ];
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  checkPhase = "true";
```

```suggestion
  dontCheck = false;
```

Why are those disabled? Please add an pythonImportsCheck instead.

---

The tests needs `ansible-playbook` to run, but if I add it, then I get:

```
Found duplicated packages in closure for dependency 'ansible':
  ansible 2.9.26 (/nix/store/w2bi455gbz60j04k72qf3d0crjnd8am7-python3.8-ansible-2.9.26/lib/python3.8/site-packages)
  ansible 3.4.0 (/nix/store/izy6iilv0iv679slin9yzipfqrrkfdlr-python3.8-ansible-3.4.0/lib/python3.8/site-packages)

Package duplicates found in closure, see above. Usually this happens if two packages depend on different version of the same dependency.
```